feature-libs/organization/administration/components/budget/services/current-budget.service.ts

```diff
@@ -32,4 +32,8 @@ export class CurrentBudgetService extends CurrentOrganizationItemService<
   protected getItem(code: string): Observable<Budget> {
     return <any>(<Budget>this.budgetService.get(code));
   }
+
+  hasError(code: string): Observable<boolean> {
```

Comment:
I wonder if the method name is ok. I'd expect here getError as well, as we don't seem to filter on whether there is an error or not.

Reply:
I agree, renamed